packages/vitest/src/types/config.ts

```diff
@@ -73,6 +73,11 @@ export interface InlineConfig {
    */
   environment?: BuiltinEnvironment
 
+  /**
+   * The configuration for the environment.
+   */
+  environmentConfiguration?: Record<string, any>
```

Comment:
```suggestion
  environmentOptions?: { jsdom?: JSdomOptions, ... }
```

Reply:
we need to check `resources` entry on `JSDOMOptions` since I don't expose the `jsdom` types.